integration-tests/test-common/src/test/java/org/apache/servicecomb/tests/tracing/TracingTestBase.java

```diff
@@ -75,6 +75,15 @@ protected void assertThatSpansReceivedByZipkin(Collection<String> logs, String..
         .map(this::extractIds)
         .collect(Collectors.toList());
 
+    // Sleep for 5 seconds to wait the reporter finish posting to zipkin server.
```

Comment:
Using the wait and try look up span id could be a reasonable way for checking the async report result.